pkg/runtime/config.go

```diff
@@ -29,6 +32,7 @@ const (
 	DefaultMetricsPort = 9090
 	// DefaultAllowedOrigins is the default origins allowed for the Dapr HTTP servers
 	DefaultAllowedOrigins = "*"
+	// DefaultAppScheme is the default URI scheme when Dapr calls the application
```

Comment:
where is `DefaultAppScheme` ? :)

Reply:
A remain from an older implementation, removed now.